vendor/tribe-selectWoo/dist/js/selectWoo.full.js

```diff
@@ -1875,7 +1875,12 @@ S2.define('select2/selection/allowClear',[
       }
     }
 
-    this.$element.val(this.placeholder.id).trigger('change');
+    // Allow clearing when the data-placeholder attribute isn't set. 
+    if ( typeof this.placeholder !== 'undefined' ) {
+      this.$element.val( this.placeholder.id ).trigger( 'change' );
+    } else {
+      this.$element.val( '' ).trigger( 'change' );
+    }
```

Comment:
Don't need to change it but think about if doing it this way makes sense:
```suggestion
    if ( typeof this.placeholder !== 'undefined' ) {
      this.$element.val( this.placeholder.id );
    } else {
      this.$element.val( '' );
    }
    
    this.$element.trigger( 'change' );
```

It reduces code duplication and makes it clear that the `if()` is just changing the value. The trigger happens either way.

Reply:
@Camwyn ooh yeah, this seems more elegant.